tests/ui/test_permissions.py

```diff
@@ -86,24 +87,76 @@ def test_has_superuser_access(app, client, user_without_role, superuser):
+    assert record_list_permission_factory().can()
 
-    assert can_list_record_factory()
-    assert can_create_record_factory()
-    assert can_update_record_factory()
-    assert can_delete_record_factory()
+    app.config.update(SONAR_APP_DISABLE_PERMISSION_CHECKS=False)
+    assert not record_list_permission_factory().can()
+
+    login_user_via_view(client, email=superuser['email'], password='123456')
+    assert record_list_permission_factory().can()
+
+
+def test_create_permission_factory(app, client, superuser, document):
+    """Test create permission factory"""
+    app.config.update(SONAR_APP_DISABLE_PERMISSION_CHECKS=True)
+    assert record_create_permission_factory(document).can()
 
     app.config.update(SONAR_APP_DISABLE_PERMISSION_CHECKS=False)
+    assert not record_create_permission_factory(document).can()
 
-    login_user_via_view(client, email=admin['email'], password='123456')
+    login_user_via_view(client, email=superuser['email'], password='123456')
+    assert record_create_permission_factory(document).can()
+
+
+def test_read_permission_factory(app, client, superuser, document):
+    """Test read permission factory"""
+    app.config.update(SONAR_APP_DISABLE_PERMISSION_CHECKS=True)
+    assert record_read_permission_factory(document).can()
+
+    app.config.update(SONAR_APP_DISABLE_PERMISSION_CHECKS=False)
+    assert not record_read_permission_factory(document).can()
+
+    login_user_via_view(client, email=superuser['email'], password='123456')
+    assert record_read_permission_factory(document).can()
+
+
+def test_update_permission_factory(app, client, superuser, document):
+    """Test update permission factory"""
+    app.config.update(SONAR_APP_DISABLE_PERMISSION_CHECKS=True)
+    assert record_update_permission_factory(document).can()
 
-    assert can_list_record_factory()
-    assert can_create_record_factory()
-    assert can_update_record_factory()
-    assert can_delete_record_factory()
-    assert can_read_record_factory(document)
+    app.config.update(SONAR_APP_DISABLE_PERMISSION_CHECKS=False)
+    assert not record_update_permission_factory(document).can()
+
+    login_user_via_view(client, email=superuser['email'], password='123456')
+    assert record_update_permission_factory(document).can()
+
+
+def test_delete_permission_factory(app, client, superuser, document):
+    """Test delete permission factory"""
+    app.config.update(SONAR_APP_DISABLE_PERMISSION_CHECKS=True)
+    assert record_delete_permission_factory(document).can()
+
+    app.config.update(SONAR_APP_DISABLE_PERMISSION_CHECKS=False)
+    assert not record_delete_permission_factory(document).can()
+
+    login_user_via_view(client, email=superuser['email'], password='123456')
+    assert record_delete_permission_factory(document).can()
+
+
+def test_unknown_permission_factory(app, client, superuser, document):
+    """Test unknown permission factory"""
+    app.config.update(SONAR_APP_DISABLE_PERMISSION_CHECKS=True)
```

Comment:
what is the idea of `SONAR_APP_DISABLE_PERMISSION_CHECKS`? Only for debug?

Reply:
Yes, it disable completely permission checks for example for testing API with external tool like Postman.

Reply:
ok